installing/installing_disconnected/installing-disconnected-preparations.adoc

```diff
@@ -1,18 +1,36 @@
 [id="installing-disconnected-preparations"]
-= Preparing to install a disconnected cluster
+= Creating a mirror registry for a disconnected cluster
 include::modules/common-attributes.adoc[]
 :context: installing-disconnected-preparations
 
 toc::[]
 
 Before you install a cluster on infrastructure that you provision in a
-disconnected environment, you must prepare the environment.
+disconnected environment, you must create a mirror registry.
 
-//include::modules/cluster-entitlements.adoc[leveloffset=+1]
+.Prerequisites
 
+* You must have access to a bastion host that can access both the internet and
+your network.
+
+include::modules/installation-about-disconnected.adoc[leveloffset=+1]
+
+[id="installing-disconnected-bastion"]
+== Prepare the bastion host
+
+Before you create the mirror registry, you must configure the bastion host.
+
+include::modules/cli-install.adoc[leveloffset=+2]
+
+include::modules/installation-creating-mirror-registry.adoc[leveloffset=+1]
+
+include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+1]
+
+include::modules/installation-mirror-repository.adoc[leveloffset=+1]
 
 .Next steps
 
+//I need to figure out how to sequence the disconnected and proxy prep.
```

Comment:
These aren't inherently ordered, you just need to setup both at some point and feed their respective settings into the `install-config.yaml`.  The only wrinkle is that if each need `additionalTrustBundle` entries, you'd need to append the second entry, but if only one or the other needs additional trust, you need to create a new `additionalTrustBundle` instead of appending.  That's not all that complicated in practice, but might require some careful wording to avoid confusion.

Reply:
I'm making a follow-up JIRA to review this sequencing after the proxy content merges: https://jira.coreos.com/browse/OSDOCS-693